gptme/tools/save.py

```diff
@@ -18,10 +20,21 @@
 If the current directory changes, the path will be relative to the new directory.
 """.strip()
 
+function_description = """
+To write to a file, use a code block with the language tag: `save <path>`
+
+The path can be relative to the current directory, or absolute.
+If the current directory changes, the path will be relative to the new directory.
+""".strip()
```

Comment:
As the comment suggests on line 15 above, we should refactor this to not mention markdown-specific instructions in the mail instructions, and move them into a separate `instructions_format` dict in the ToolSpec.

Reply:
Did it.